src/tinymce.js

```diff
@@ -64,10 +67,10 @@ angular.module('ui.tinymce', [])
           return function(ed) {
 	        $timeout.cancel(debouncedUpdateTimer);
 	         debouncedUpdateTimer = $timeout(function() {
-              return (function(ed) {
-                if (ed.isDirty()) {
-                  ed.save();
-                  updateView(ed);
+              return (function(debouncedEditor) {
```

Comment:
Regarding this IIFE, you asked what was the value of it. So I'm guessing leaving it here to prevent breaking stuff, one change at a time?

Reply:
Yep, I decided to leave it as we currently have debounce set to false so bypass anyway- so limiting the area I'm touching to avoid potential sadness